.github/workflows/linux.yml

```
@@ -301,7 +301,8 @@ jobs:
           sudo incus exec target \
             -- \
             /host/packages/${PACKAGE_TYPE}/test.sh \
-              ${{ matrix.package }}-mroonga
+              ${{ matrix.package }}-mroonga \
+              ${{ github.ref_type }}
```

環境変数でいけるところは環境変数を使ってください。`${{ ... }}`だと2段階で評価される（シェルスクリプト実行前に`${{ ... }}`が評価されて、シェルスクリプト実行時に`github.ref_type`の中身が評価される）ので理解しにくくなります。

```suggestion
              ${GITHUB_REF_TYPE}
```

---

fix: [aa1cd4c](https://github.com/mroonga/mroonga/pull/876/commits/aa1cd4c3ecb1c2e6db86a765c4296d2cd36f41fc) たしかにです。了解です！